src/test/java/org/zalando/riptide/AsyncTest.java

```diff
@@ -75,6 +75,20 @@ public void shouldCall() {
     }
 
     @Test
+    public void shouldCallWithoutParameters() {
```

Comment:
Because this was the only point I've found the other untyped `call` method being tested.